lib/src/_shared/ModalDialog.tsx

```diff
@@ -38,76 +63,56 @@ export const ModalDialog: React.SFC<ModalDialogProps & WithStyles<typeof styles>
   showTabs,
   wider,
   ...other
-}) => (
-  <Dialog
-    role="dialog"
-    onClose={onDismiss}
-    classes={{
-      paper: clsx(classes.dialogRoot, {
-        [classes.dialogRootWider]: wider,
-      }),
-    }}
-    {...other}
-  >
-    <DialogContent children={children} className={classes.dialog} />
-
-    <DialogActions
+}) => {
+  const classes = useStyles();
+  return (
+    <Dialog
+      role="dialog"
```

Comment:
Is the role needed?

Reply:
Haha, this line was added more than 2 years ago, sure it is not needed